### PR TITLE
Hyperlink to the inputs context documentation

### LINKED ISSUE #24243: Provide a link to the context documentation on inputs when relevant
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_callinputs

### What part(s) of the article would you like to see updated?

When the contexts subject is referred to, it would be helpful if the relevant documentation article is hyperlinked. I'll open a PR shortly that does this.

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by gbakeman**
Provide a link to the contexts documentation for additional information about inputs.

### Why:
It would be helpful if the contexts documentation was linked directly when it's referenced under the documentation about inputs.

Closes #24243

### What's being changed (if available, include any code snippets, screenshots, or gifs):
Simply hyperlinking the text that reads `inputs context`

### Check off the following:

- [x] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by cmwilson21** [comment]
@gbakeman Thank you for opening a PR and linking it to your issue! ✨ 

I'll get this triaged for review! :zap:

**Comment by gbakeman** [comment]
Thank you @cmwilson21 !

File: content/actions/using-workflows/workflow-syntax-for-github-actions.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ In addition to the standard input parameters that are available, `on.workflow_ca
 
 If a `default` parameter is not set, the default value of the input is `false` for a boolean, `0` for a number, and `""` for a string.
 
-Within the called workflow, you can use the `inputs` context to refer to an input.
+Within the called workflow, you can use the [`inputs` context](/actions/learn-github-actions/contexts#inputs-context) to refer to an input.
```
Comment (SiaraMist) [inline content/actions/using-workflows/workflow-syntax-for-github-actions.md:85]:
```suggestion
Within the called workflow, you can use the `inputs` context to refer to an input. For more information, see "[AUTOTITLE](/actions/learn-github-actions/contexts#inputs-context)."
```
Just updating the link to match our style guide. 👍

**Review by SiaraMist — DISMISSED** [review]
Hi @gbakeman, thank you so much for your contribution to the docs! 

I agree that linking to the reference docs for the `inputs` context would be really helpful here. I just updated the link formatting to match our [docs style guide](https://github.com/github/docs/blob/main/contributing/content-style-guide.md#links), and now I'll go ahead and get this merged! 🚀

**Review by SiaraMist — APPROVED** [review]

**Comment by gbakeman** [comment]
@SiaraMist Sorry for the additional work to correct the link - lesson learned though, and I hope I can contribute in the future. Thank you for accepting my change.

**Comment by SiaraMist** [comment]
Oh no worries at all, I wouldn't expect you to memorize our style guide before opening a PR. 😅 Thank you again for improving the docs! 💛

**Comment by Theinzawok** [comment]
နမူနာဖိုင်လေးပေးပါလားခင်ဗျ

**Pull request merged**